DependencyInjection/DoctrinePHPCRExtension.php

```diff
@@ -194,6 +198,12 @@ private function loadJackalopeSession(array $session, ContainerBuilder $containe
                 $connectionAliasName = sprintf('doctrine_phpcr%s.jackalope_doctrine_dbal.%s_connection', $serviceNamePrefix, $session['name']);
                 $container->setAlias($connectionAliasName, $connectionService);
 
+                // If default connection does not exist set the first doctrine_dbal connection as default
+                $defaultConnectionAlias = sprintf('doctrine_phpcr%s.jackalope_doctrine_dbal.%s_connection', $serviceNamePrefix, 'default');
```

Comment:
no need to sprintf for the 'default' connection name, as its a constant anyways. lets simplify to `sprintf('doctrine_phpcr%s.jackalope_doctrine_dbal.default_connection', $serviceNamePrefix);`